File: Mobile/app/src/main/java/com/fivegearszerochill/paladins/domain/models/player/MatchHistoryModel.kt
package com.fivegearszerochill.paladins.domain.models.player

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class MatchHistoryModel(
    @Expose
    @SerializedName("id")
    val id: Long? = null,
    @Expose
    @SerializedName("paladinsMatchId")
    val paladinsMatchID: Long? = null,
    @Expose
    @SerializedName("paladinsChampionId")
    val paladinsChampionID: Long? = null,
    @Expose
    @SerializedName("championName")
    val championName: String? = null,
    @Expose
    @SerializedName("kills")
    val kills: Long? = null,
    @Expose
    @SerializedName("deaths")
    val deaths: Long? = null,
    @Expose
    @SerializedName("assists")
    val assists: Long? = null,
    @Expose
    @SerializedName("matchResult")
    val matchResult: String? = null
){
    override fun toString(): String {
        return "MatchHistoryModel(id=$id, paladinsMatchID=$paladinsMatchID, paladinsChampionID=$paladinsChampionID, championName=$championName, kills=$kills, deaths=$deaths, assists=$assists, matchResult=$matchResult)"
    }
}

File: Paladins.Api/Paladins.Api/Paladins.Common/Models/MatchHistoryModel.cs
using Paladins.Common.Interfaces.Definitions;

namespace Paladins.Common.Models
{
    public class MatchHistoryModel : IModel
    {    
        public int Id { get; set; }
        public int PaladinsChampionId { get; set; }
        public int PaladinsMatchId { get; set; }
        public string ChampionName { get; set; }
        public int Kills { get; set; }
        public int Deaths { get; set; }
        public int Assists { get; set; }
        public string MatchResult { get; set; }
    }
}
